index.html

```diff
@@ -140,11 +115,15 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 			$.getJSON('https://api.github.com/orgs/heremaps/repos')
 			.done(function (repo_data) {
 				repo_data.forEach(function (data, index) {
+					/* The GitHub API will return more repos than the ones we expose here.
+						 Target only existing repo sections.*/
 					var $target = $('#repo-info-' + data.name);
 					if ($target.length > 0) {
-						$target.find('[data-role="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-role="fork-count"]').text(data.forks_count);
-						$target.find('[data-role="star-count"]').text(data.stargazers_count);
+						$target.find('[data-key="repo-type"]').text(data.language);
```

Comment:
I didn't know about GitHub linguist's extensions to `.gitattributes`, that looks doable. However, at the example of https://github.com/heremaps/dita-ot-plugins I'm not sure how the path pattern should look like as we'd need to refer to submodule paths, which are not really "there" in the Git repository. If it still works by referring to the submodule path names, I'll all in for that solution.